src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepositorySpec.scala

```diff
@@ -195,4 +195,42 @@ class FileMetadataRepositorySpec extends AnyFlatSpec with TestDatabase with Scal
     fileTwoMetadata.get.head.propertyname should equal("FilePropertyOne")
     fileTwoMetadata.get.head.value should equal("Result of FileMetadata processing")
   }
+
+  "getClosureMetadataProperty" should "return the correct closure metadata property" in {
+    val db = DbConnection.db
+    val fileMetadataRepository = new FileMetadataRepository(db)
+    deleteTables()
```

Comment:
Is there a reason for deleting the tables in this test and not the other tests that have been added?

Reply:
Somewhere in the code the FilePropertyTable was being used which was causing this test to fail with an extremely vague error. @MancunianSam said at some point we are gonna remove the h2 database so most of the test will need to be redone anyway. unless I am mistaken.